web/js/components/image-download/lat-long-inputs.js

```diff
@@ -26,9 +26,13 @@ function Input({
       const newInputValue = Number(inputValue);
       const newArray = structuredClone(boundingBoxArray);
```

Comment:
I'm nitpicking here but I really dislike the name "newArray". Can we come up with something more descriptive?